components/QRCodeComponent.tsx

```diff
@@ -56,9 +54,7 @@ const menuActions: Action[] =
 
 const QRCodeComponent: React.FC<QRCodeComponentProps> = ({
```

Comment:
This component is used in a lot of places, the usage of this component would also need to be updated.